src/main/java/com/google/devtools/build/lib/exec/SpawnStrategyRegistry.java

```diff
@@ -129,7 +132,7 @@ public List<? extends SpawnStrategy> getStrategies(Spawn spawn, @Nullable EventH
   public List<? extends SpawnStrategy> getStrategies(
       ActionExecutionMetadata resourceOwner, String mnemonic, @Nullable EventHandler reporter) {
     // Don't override test strategies by --strategy_regexp for backwards compatibility.
-    if (!"TestRunner".equals(mnemonic)) {
```

Comment:
Since this is for backwards compatibility only, maybe we can drop it now or at least keep it restricted to actual test spawns, no postprocessing?

Reply:
Yeah that makes sense. Will restrict this to TestRunner only